DataManagementSystem/Client/FTS3Job.py

```diff
@@ -93,8 +94,7 @@ def monitor( self, context = None, ftsServer = None, ucert = None ):
     if not context:
       if not ftsServer:
         ftsServer = self.ftsServer
-      context = fts3.Context( endpoint = ftsServer, ucert = ucert )
-
+      context = fts3.Context(endpoint=ftsServer, ucert=ucert, request_class=ftsSSLRequest, verify=False)
```

Comment:
you may want to add the same comment about the bug in pycurl that there is in the FTSAgent